static/app/components/feedback/feedbackItem/feedbackScreenshot.tsx

```diff
@@ -28,23 +30,29 @@ export default function FeedbackScreenshot({
         if (!hasRole) {
           return null;
         }
+
+        const img = (
+          <StyledImageVisualization
+            attachment={screenshot}
+            orgId={organization.slug}
+            projectSlug={projectSlug}
+            eventId={screenshot.event_id}
+            onLoad={() => setIsLoading(false)}
+            onError={() => setIsLoading(false)}
+          />
+        );
         return (
-          <StyledPanel>
+          <StyledPanel className={className}>
             {isLoading && (
               <StyledLoadingIndicator>
                 <LoadingIndicator mini />
               </StyledLoadingIndicator>
             )}
-            <StyledImageButton onClick={onClick}>
-              <StyledImageVisualization
-                attachment={screenshot}
-                orgId={organization.slug}
-                projectSlug={projectSlug}
-                eventId={screenshot.event_id}
-                onLoad={() => setIsLoading(false)}
-                onError={() => setIsLoading(false)}
-              />
-            </StyledImageButton>
+            {onClick ? (
```

Comment:
why did we extract out the `onClick` here?

Reply:
I'm re-using FeedbackScreenshot within the modal.

It's got the loading indicator thing and the rounded-corner styles.
Also, as i was typing this i realized it has the `<Role>` check, which should really go around the whole `<ScreenshotSection>` including the delete buttons! So that got moved up the tree a little bit.